manifests/security/nodeagent/templates/daemonset.yaml

```diff
@@ -34,7 +34,11 @@ spec:
 {{- else }}
           image: "{{ .Values.nodeagent.hub | default .Values.global.hub }}/{{ .Values.nodeagent.image }}:{{ .Values.nodeagent.tag | default .Values.global.tag }}"
 {{- end }}
-          imagePullPolicy: {{ .Values.global.imagePullPolicy | default "Always" }}
+{{- if or (contains "dev" .Values.global.nodeagent.image) (contains "dev" .Values.nodeagent.tag) (contains "dev" .Values.global.tag) }}
```

Comment:
I think it may be better to just have the 
```
{{ if .Values.global.imagePullPolicy }}
          imagePullPolicy: {{ .Values.global.imagePullPolicy }}
{{- end }}
```

to keep things simpler, otherwise we run into confusing behavior

Reply:
ok i'll make the change, you are the one who suggested dev :)